xc/xenops_server_xen.ml

```diff
@@ -2940,15 +2940,11 @@ module VIF = struct
               let open Xenops_interface.Pci in
               let pci_state = PCI.get_state' vm pci in
               let open Xenops_interface.Pci in
```

Comment:
Remove this redundant line

Reply:
Done.  Thanks for this comment!
Terrible error really.